playbooks/openshift/pre-install.yml

```diff
@@ -1,70 +1,8 @@
 ---
-# Assign hostnames
-- hosts: cluster_hosts
-  pre_tasks:
-  - include: roles/common/pre_tasks/pre_tasks.yml
-  roles:
-  - role: hostnames
-
-# Subscribe Hosts
```

Comment:
The host/instance used for DNS needs to be subscribed in order to set up the necessary RPMs use by bind/named. Hence, this needs to happen before the DNS steps - i.e.: same order as before. We can, however, look into splitting them out and do a subscription just for the DNS instance here, and allow for a generic subscription approach in the pre-install steps still.